Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/MarkdownTextBlockPage.xaml.cs

```diff
@@ -69,7 +70,14 @@ private void SetInitalText(string text)
 
         private async void MarkdownText_LinkClicked(object sender, UI.Controls.LinkClickedEventArgs e)
         {
-            await Launcher.LaunchUriAsync(new Uri(e.Link));
+            if (e.Link.StartsWith("../") || e.Link.StartsWith("/"))
```

Comment:
Should put the # here as well, maybe we should have a helper method for this?

Reply:
Where do we put all Helper Methods? I see `Common.cs`. Should I use that?

Also i see Url's being handled this way in `HyperlinkInline.cs`. I can change it over there also if `Common.cs` is the right place.

Reply:
Well, Common is for Markdown still, and the e-mail regex is in the TextBoxRegEx control as well.

@nmetulev would adding an extensions namespace to [Microsoft.Toolkit.Uwp](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp) make sense here or use the existing extensions namespace under `Microsoft.Toolkit.Uwp.UI`?

Reply:
What's the helper method here?

Reply:
@nmetulev `IsUriRelative` to go with the `IsEmail` below.  Thoughts on the namespace choice?

Reply:
Is it used outside of the controls package? If it's only used here, I'd try to keep it internal.

Reply:
@nmetulev Issue here is it cannot be internal because the same Link check is Happening on SampleApp Project. Only way we can do this is to move this to Microsoft.Toolkit.UWP.UI so that it can be used in both places.

Reply:
Ah, i missed that this is in the sample app. Why not use the standard way of checking for a relative URI?

`Uri.TryCreate(url, UriKind.Absolute, out Uri result)`

Reply:
Right now for relative URI's we only check if URL's start with `/`, '#', `../`. If we check for the above, we are practically allowing anything. I don't think that is a good idea.

Reply:
Exactly, I like leaving it up to the developer to decide what to check, so I'm good with the current implementation :)